src/org/zaproxy/zap/extension/stdmenus/ExtensionStdMenus.java

```diff
@@ -179,6 +186,23 @@ public void actionPerformed(java.awt.event.ActionEvent e) {
 		}
 		return popupContextTreeMenuOutScope;
 	}
+	
+	private PopupMenuExportContextURLs getPopupContextTreeMenuExportUrls() {
+		if (popupContextTreeMenuExportUrls == null) {
+			popupContextTreeMenuExportUrls = new PopupMenuExportContextURLs(
+					Constant.messages.getString("context.export.urls.menu")) {
+				private static final long serialVersionUID = 1L;
+
+				@Override
+				public boolean isEnableForComponent(Component invoker) {
+					return invoker.getName().equals(SiteMapPanel.CONTEXT_TREE_COMPONENT_NAME);

```

Comment:
Worth also checking that there's a node selected and it's not root (like other context pop up menu items do).